packages/authentication/types/user.ts

```diff
@@ -5,19 +5,31 @@ export interface User {
   newEmail: string
   isNewEmailConfirmed: boolean
   referralCode: string
-  avatar: {
-    fullSize: string
-    small: string
-  }
-  firstName: string
-  lastName: string
+  profile: MinimalProfile
   phoneNumber: string
   preferredLanguage: string
 }
 
+export interface MinimalProfile {
```

Comment:
Should I move this elsewhere? I did not want to put it on `components/modules/profiles` because the `authentication` package should probably not depend on the `components` package. But maybe to `utils/types`?